### PR TITLE
feat(simulator): show a message when the CC is not supported

### LINKED ISSUE #3543: Ajouter modale "pas de CC" et message "CC non prise en compte" outil préavis retraite
**Problème**
Simulateur départ ou mise à la retraite : étape "convention collective"
1)  Actuellement il n'y a pas de message indiquant à l'utilisateur qu'il n'a pas choisi de CC et donc qu'il peut continuer mais que sa CC peut prévoir un délais plus favorable
2) Actuellement il n'y a pas de message indiquant à l'utilisateur que la CC indiquée n'est pas traitée par nos services lors du choix de la CC

**Solution**
1) Si l'utilisateur clique sur "suivant" sans avoir rempli de CC le modal suivant apparaît:

Group 2

2) Si l'utilisateur sélectionne une CC qui n'est pas traitée : 

Group 5

https://www.figma.com/file/8drdd9OhCJjYqffvYeJynl/Outil-pr%C3%A9avis-de-retraite?node-id=36%3A0

### PR DESCRIPTION
**Pull request by m-maillot**
Closes #3543

### CODE REVIEW

File: packages/code-du-travail-modeles/src/build.ts [hunk 1]
```diff
@@ -33,5 +35,21 @@ function copyJSFile() {
   fse.copy(`${inDir}/utils`, `${outDir}/utils`);
 }
 
+function writeSupportedCCFile() {
+  const modeles = mergeModels();
+  const ccn = extractImplementedCc(new Engine(modeles));
+  const jsString =
+    '"use strict";\n' +
```
Comment (lionelB) [inline packages/code-du-travail-modeles/src/build.ts:42]: Could it be a json file ?

File: packages/code-du-travail-modeles/src/__test__/extract.implemented.cc.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import Engine from "publicodes";
+import { mergeModels } from "../internal/merger";
+import { extractImplementedCc } from "../internal/ExtractSupportedCc";
+
+const engine = new Engine(mergeModels());
+
+test("Extract all supported CC from the YAML modeles", () => {
+  const result = extractImplementedCc(engine);
+
+  expect(result).toContain(expect.arrayContaining([44, 176, 650]));
```
Comment (lionelB) [inline packages/code-du-travail-modeles/src/__test__/extract.implemented.cc.spec.tsx]: where does these number comme from ? We have implemented more ccn

File: packages/code-du-travail-modeles/src/build.ts (hunk 1 above)
Comment (m-maillot) [reply packages/code-du-travail-modeles/src/build.ts:42]: I prefer using typescprit here to have properties types

File: packages/code-du-travail-modeles/src/__test__/extract.implemented.cc.spec.tsx (hunk 2 above)
Comment (m-maillot) [reply packages/code-du-travail-modeles/src/__test__/extract.implemented.cc.spec.tsx]: Done :)

**Review by lionelB — APPROVED** [review]

**Pull request merged**